BackofficeBundle/Resources/public/coffee/FullPageFormView.coffee

```diff
@@ -5,7 +5,7 @@ FullPageFormView = OrchestraView.extend(
     @options = options
     @options.listUrl = appRouter.generateUrl('listEntities', entityType: options.entityType) if options.listUrl == undefined
     @completeOptions(@options.element) if @options.element != undefined
-    @events = {}
+    @events = @events || {}
```

Comment:
I extend because of extendView. If i create a FullPageFormView with extendView generateId (TableviewCollectionView line 88), the orchestraView merge the events of generateId, call the initialize which erase this events, and delegate the events. Too late, no more event! @nicolasThal